lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/MessagingTransportFactory.java

```diff
@@ -52,14 +51,6 @@ public MessagingTransportFactory(final LocalAddressProvider localAddressProvider
   }
```

Comment:
It's used within a non-deprecated public constructor in `NameLookupClient` and a post-0.11 deprecated constructor in `NameClient`. We would have to remove these constructors, and then recursively see if they're used in other places as well.